src/windows.d.ts

```diff
@@ -1,7 +1,12 @@
 import { ILiveConnect } from './types'
 
 declare global {
+  type XDomainRequest = any
+
   interface Window {
+    // eslint-disable-next-line camelcase
     liQ_instances?: ILiveConnect[]
+    liQ?: ILiveConnect
+    XDomainRequest?: { new (): XDomainRequest; prototype: XDomainRequest; create(): XDomainRequest; };
```

Comment:
was it required to define `XDomainRequest` ?
it sounds interesting because in [liveconnectbuilder](https://github.com/LiveIntent/liveconnectbuilder/blob/master/src/utils/calls.js#L33) the `calls.js` is using it and I couldn't find from where it will be initialized